demos/browser/app/messagingSession/messagingSession.ts

```diff
@@ -19,6 +19,7 @@ import {
 } from 'amazon-chime-sdk-js';
 
 import { ChimeSDKMessagingClient, GetMessagingSessionEndpointCommand } from '@aws-sdk/client-chime-sdk-messaging';
+import PrefetchOn from "amazon-chime-sdk-js";
```

Comment:
You can just add the `PrefetchOn` to lines 9-18 above with existing imports from JS SDK. Also, we use single quote export over double for convention and consistency.